libs/dal/src/repositories/notification-template/notification-template.repository.ts

```diff
@@ -8,7 +8,12 @@ import { DalException } from '../../shared';
 import type { EnforceEnvOrOrgIds } from '../../types/enforce';
 import { EnvironmentRepository } from '../environment';
 
-type NotificationTemplateQuery = FilterQuery<NotificationTemplateDBModel> & EnforceEnvOrOrgIds;
+type NotificationTemplateByIdQuery = FilterQuery<NotificationTemplateDBModel> & EnforceEnvOrOrgIds;
```

Comment:
Why did we rename this? It's not just by ID.

Reply:
it was an internal problematic object this filterQuery, I added a real interface exposed and renamed the internal one